tests/test_handler_checkpoint_saver.py

```diff
@@ -100,6 +100,26 @@ def _train_func(engine, batch):
             self.assertTrue(os.path.exists(os.path.join(tempdir, filename)))
         shutil.rmtree(tempdir)
 
+    def test_exception(self):
+        logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+        net = torch.nn.PReLU()
+        tempdir = tempfile.mkdtemp()
```

Comment:
I might have forgotten if we've discussed it but is there a reason not to use `tempfile.TemporaryDirectory` here?

Reply:
The reason is that: `tempfile.TemporaryDirectory` uses current system temp folder and it's not easy to delete all the generated files after executing tests, if use `tempfile.mkdtemp()`, we can delete the temp folder directly.

Thanks.

Reply:
When the `TemporaryDirectory` context exits it deletes the directory and contents. The advantage is that cleanup is guaranteed even if the block within the context raises an exception.

```python
with tempfile.TemporaryDirectory() as tmpdirname:
    print(tmpdirname)
    with open(tmpdirname+'/test.txt','w') as o:
        o.write('test')
        
print(os.path.exists(tmpdirname))  # False
```

Reply:
OK, I see your point. Updated in the latest commit.
And I will file a separate PR to update all the other tests that use `tempfile.mkdtemp()`.
I submitted issue https://github.com/Project-MONAI/MONAI/issues/895 to track this task.

Thanks.